index.html

```diff
@@ -1,58 +1,374 @@
+            <div class="categories">
+              <div class="categories-header-container">
+                <img src="./images/language-icon.svg" alt="Diamond Icon" />
+                <h3 class="categories-header">Language</h3>
+              </div>
+              <ul class="categories-list">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </div>
+
+            <div class="categories">
+              <div class="categories-header-container">
+                <img src="./images/framework-icon.svg" alt="Rectangle Icon" />
+                <h3 class="categories-header">Frameworks</h3>
+              </div>
+              <ul class="categories-list">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+
+            <div class="categories">
+              <div class="categories-header-container">
+                <img src="images/skills-icon.svg" alt="round icon" />
+                <h3 class="categories-header">Skills</h3>
+              </div>
+              <ul class="categories-list">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+            </div>
+          </div>
+        </section>
+
+        <section id="contact-me">
+          <div class="contact-info">
+            <h2>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h2>
+          </div>
+
+          <form
+            class="contact-form-container"
+            id="contact-form"
+            action="https://formspree.io/f/xrgdbeyr"
+            method="post"
+          >
+            <ul>
+              <li>
+                <input
+                  type="text"
+                  id="name"
+                  name="Name"
+                  placeholder="Full name"
+                  required
+                  maxlength="30"
+                />
+              </li>
+              <li>
+                <input
+                  type="email"
+                  id="email"
+                  name="Email"
+                  placeholder="Email address"
+                  required
+                />
+              </li>
+              <li>
+                <textarea
+                  name="Text Area"
+                  id="textarea"
+                  placeholder="Enter text here"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </li>
+              <li>
+                <button type="submit">Get in touch</button>
+              </li>
+            </ul>
+          </form>
+        </section>
```

Comment:
- [ ] Kindly use `flexbox` to layout the elements in this `section` or for `form` only. It's very important to learn how to use `flexbox` in CSS as it makes elements `flexible` and more `responsive`.
***One of the main advantages of flexbox is the ability to fill extra space without the need to use Javascript.***
- Here is an example of using flexbox for `form` instead of `ul` and `li`.

```
form {
    display: flex:
    flex-direction: column;
}
```